app/src/newdocs/components/CanvasModuleConfig/index.jsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable flowtype/no-types-missing-file-annotation */
+import React from 'react'
+import isEmpty from 'lodash/isEmpty'
+import ReactMarkdown from 'react-markdown'
+
+import { createMdSnippet } from '$newdocs/components/utils'
+
+const inputs = (moduleInputs) => (
+    !isEmpty(moduleInputs)
+        ? createMdSnippet(moduleInputs)
+        : false
+)
+
+const outputs = (moduleOutputs) => (
+    !isEmpty(moduleOutputs)
+        ? createMdSnippet(moduleOutputs)
+        : false
+)
+
+const params = (moduleParams) => (
+    !isEmpty(moduleParams)
+        ? createMdSnippet(moduleParams)
+        : false
+)
+
+type Props = {
+    moduleInputs: any,
+    moduleOutputs: any,
+    moduleParams: any
+}
+
+const CanvasModuleConfig = ({ moduleInputs, moduleOutputs, moduleParams }: Props) => (
+    <React.Fragment>
+        {!isEmpty(moduleInputs) ? (
+            <React.Fragment>
+                <strong>
+                    Inputs
+                </strong>
+                <ReactMarkdown source={inputs(moduleInputs)} />
+            </React.Fragment>
+        ) : ''}
```

Comment:
Generally best to use `null` instead of `''`, empty string creates an unnecessary empty Text node:

e.g. https://codesandbox.io/embed/eloquent-sea-sw471

![image](https://user-images.githubusercontent.com/43438/62270628-03cf6500-b469-11e9-800e-23d71fdf37bc.png)

![image](https://user-images.githubusercontent.com/43438/62270616-00d47480-b469-11e9-8a74-bd17840545a2.png)